src/App.php

```diff
@@ -1100,14 +1100,24 @@ protected function outputResponse(string $data, array $headers): void
 
         foreach (ob_get_status(true) as $status) {
             if ($status['buffer_used'] !== 0) {
-                throw new LateOutputError('Unexpected output detected');
+                $lateError = new LateOutputError('Unexpected output detected');
+                if ($this->catch_exceptions) {
+                    $this->caughtException($lateError);
```

Comment:
should terminate (and if not overrode it always will), if it does not, throw line below save us